sklearn/svm/tests/test_sparse.py

```diff
@@ -323,6 +325,8 @@ def test_sparse_liblinear_intercept_handling():
     test_svm.test_dense_liblinear_intercept_handling(svm.LinearSVC)
 
 
+# TODO(1.8): remove warning filter when setting gamma raises an error
+@pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
shouldn't we be fixing the test so that no warning is raised?

Reply:
> shouldn't we be fixing the test so that no warning is raised?

Yes adapting the tests to these new changes sounds good to me